packages/orchestrator/claim_schema.py

```
@@ -0,0 +1,221 @@
+class Vote:
+    """A single voter's ternary evaluation of a Claim."""
+
+    voter: str
+    vote: int  # -1, 0, +1
+    rationale: str
+    voted_at: str = field(default_factory=_utcnow_iso)
+
+    def validate(self) -> None:
+        """Enforce spec invariants INV-002, INV-005."""
+        if self.vote not in (-1, 0, 1):
+            raise ValueError(f"E_VOTE_INVALID_RANGE: {self.vote} not in (-1, 0, +1)")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)